app/views/spree/taxons/_sorting_filters.html.erb

```diff
@@ -1,18 +1,18 @@
-
 <ul class="" style="display:none;">
   <li><%#= link_to "Most Popular", request.parameters.merge(sorting: :descend_by_popularity), id: 'descend_popularity',class: "most-popular" %></li>
   <li><%= link_to "Ascending name", request.parameters.merge(sorting: :ascend_by_name), id: 'ascending_name',class: "assending-filter" %></li>
   <li><%= link_to "Descending name", request.parameters.merge(sorting: :descend_by_name), id: 'descending_name', class: "descending-filter" %></li>
+  <li><%= link_to "bestsellers", request.parameters.merge(sorting: :bestsellers), id: 'bestsellers_filter', class: "bestsellers-filter" %></li>
 </ul>
 
 <div class="sorting-filter d-flex align-items-center justify-content-end" >
   <label>Sort by:</label>
   <div class="product-sort-js" data-filter-type="most-popular">
+    <% active_klass_descending = "" %>
```

Comment:
Instead of assign the value twice to this variable, you can do this in single line like this:

active_klass_descending = params[:sorting].present? && params[:sorting] == "descend_by_popularity" ? 'active' : ''

 Now you can call this variable inside class